mayhem/test_fuzz.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
```

Comment:
Cna we rename this to explain what's acutally being fuzzed? "test_fuzz" is not very descriptive.

Reply:
I have renamed this to 'nonsense_fuzz' as it is fuzzing the 'nonsense' function of this library.